v2/bindings/csharp/SnapPluginLibCs/CollectorExample/Program.cs

```diff
@@ -1,4 +1,7 @@
 ﻿using System;
+using Autofac;
```

Comment:
the goal is to have none (if possible) dependencies etc. Even if this is example.

Reply:
?? In C# world using directive allow the use of types in a namespace so that you do not have to qualify the use of a type in that namespace 
-> https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-directive